tests/basic/volume_ops_test.go

```diff
@@ -1466,3 +1469,136 @@ var _ = Describe("{ResizeVolumeAfterFull}", func() {
 	})
 
 })
+
+var _ = Describe("{CreateFastpathVolumeRebootNode}", func() {
+	var testrailID = 0
+	// JIRA ID : https://portworx.atlassian.net/browse/PTX-15700
+	var runID int
+	JustBeforeEach(func() {
+		StartTorpedoTest("CreateFastpathVolumeRebootNode",
+			"Create fast path volume, reboot the node, check fastpath is active", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+		log.Infof("The runID  %v ", runID)
+	})
+
+	var pxNode node.Node
+	var contexts []*scheduler.Context
+	var applist = Inst().AppList
+	stepLog := "Create fastpath Volume Reboot Node and check if fastpath is active"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+		stepLog = "Step 1: Find all the Storage nodes and and select a node for test"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			// Get all the Nodes
+			pxNodes, err := GetStorageNodes()
+			log.FailOnError(err, "Unable to get the storage nodes")
+			log.Infof("The pxNodes %v ", pxNodes)
+
+			// Select the first node for the test
+			pxNode = pxNodes[0]
+			log.Infof("The Selected node as Fast path label is %v : ", pxNode)
+
+			// Remove if node-type label is set before the test
```

Comment:
can we move it to common.go ? and pass labels to remove as argument ? so that any label specified can be removed ?